### PR TITLE
euroradio.fm

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by fuglede** [comment]
Lots of (false) mixed content blocking on this one.

**Comment by galeksandrp** [comment]
Added `platform="mixedcontent"`

File: src/chrome/content/rules/euroradio.fm.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+m.euroradio.fm ¹
```
Comment (J0WI) [inline src/chrome/content/rules/euroradio.fm.xml]: ihwc2015.euroradio.fm

Comment (galeksandrp) [inline src/chrome/content/rules/euroradio.fm.xml]: Added.

**Pull request merged**